packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -5,10 +5,10 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.9
+    tag: v1.0.10-rc2
   securityScan:
     repository: rancher/security-scan
-    tag: v0.2.8
+    tag: v0.2.9-rc6
```

Comment:
@rayandas we should bump the images for rancher/cis-operator and  rancher/security-scan to non-rc versions now that 2.7.0 is out

Reply:
Ah! my bad. Bumped the versions.